website/docs/reference/faq.md

```diff
@@ -97,3 +97,30 @@ Link to support page
     Keep in mind that the ports will increment as you connect multiple devices.
   </div>
 </details>
+
```

Comment:
Below should not be under the `## Using the REPL` section. Make a new section, maybe `## General` and move it above  `## Using the REPL